src/Lykke.Service.PayPushNotifications/Modules/ServiceModule.cs

```diff
@@ -0,0 +1,94 @@
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            // Do not register entire settings in container, pass necessary settings to services which requires them
+            //builder.RegisterInstance(_log)
+            //    .As<ILog>()
+            //    .SingleInstance();
+
+            var mapperProvider = new MapperProvider();
+            IMapper mapper = mapperProvider.GetMapper();
+            builder.RegisterInstance(mapper).As<IMapper>();
+
+            builder.RegisterInstance<IEmployeeNotificationIdRepository>(
+                new EmployeeNotificationIdRepository(
+                    AzureTableStorage<EmployeeNotificationIdEntity>.Create(
+                        _appSettings.ConnectionString(x => x.PayPushNotificationsService.Db.DataConnString),
+                        _appSettings.CurrentValue.PayPushNotificationsService.Db.EmployeeNotificationIdsTableName,
+                        _logFactory)));
+
+
+            builder.RegisterInstance<IMerchantNotificationIdRepository>(
+                new MerchantNotificationIdRepository(
+                    AzureTableStorage<MerchantNotificationIdEntity>.Create(
+                        _appSettings.ConnectionString(x => x.PayPushNotificationsService.Db.DataConnString),
+                        _appSettings.CurrentValue.PayPushNotificationsService.Db.MerchantNotificationIdsTableName,
+                        _logFactory)));
+
+            builder.RegisterInstance<INotificationRepository>(
+                new NotificationRepository(
+                    AzureTableStorage<NotificationEntity>.Create(
+                        _appSettings.ConnectionString(x => x.PayPushNotificationsService.Db.DataConnString),
+                        _appSettings.CurrentValue.PayPushNotificationsService.Db.NotificationsTableName, _logFactory)));
+
+            builder.RegisterType<PayLoadBuilderFactory>()
+                .As<IPayLoadBuilderFactory>()
+                .SingleInstance();
+
+            builder.RegisterType<NotificationClientFactory>()
+                .As<INotificationClientFactory>()
+                .SingleInstance()
+                .WithParameter("clients",
+                    new Dictionary<NotificationPlatform, INotificationClient>
+                    {
+                        {
+                            NotificationPlatform.Aps,
+                            AppleNotificationClient.CreateClientFromConnectionString(
+                                _appSettings.CurrentValue.PayPushNotificationsService.HubConnectionString,
+                                _appSettings.CurrentValue.PayPushNotificationsService.HubName)
+                        }
+                    });
+
+            builder.RegisterType<NotificationService>()
+                .As<INotificationService>()
+                .SingleInstance();
+
+            builder.RegisterType<NotificationSubscriber>()
+                .As<IStartable>()
```

Comment:
I would not recommend registering it as IStartable. In this case service startup time can be increased. Instead, register it as IStopable only, remove autoactivation and start it manually in StartupManager.